packages/blade/src/components/Counter/counterTokens.ts

```diff
@@ -0,0 +1,27 @@
+import type { CounterProps } from './Counter';
+import type { TypographyPlatforms } from '~tokens/global/typography';
+import type { DotNotationSpacingStringToken } from '~src/_helpers/types';
+
+type BadgeMaxWidth = 100 | 120;
```

Comment:
You probably mean:

```suggestion
type CounterMaxWidth = 100 | 120;
```

Reply:
haha yeah

Reply:
@anuraghazra You suck at doing a copy, paste. I can imagine how cheating in an exam would be risky with you! ;)